Snowflake.Data.Tests/UnitTests/ArrowResultChunkTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿/*
+ * Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+using System;
+using System.Linq;
+using Apache.Arrow;
+using NUnit.Framework;
+using Snowflake.Data.Core;
+
+namespace Snowflake.Data.Tests.UnitTests
+{
+    [TestFixture]
+    class ArrowResultChunkTest
+    {
+        private static int s_rowCount = 10;
+        private RecordBatch _recordBatch;
+        private ArrowResultChunk _chunk;
+        
+        [Test]
+        [Ignore("ArrowResultChunkTest")]
+        public void SFArrowResultChunkTestDone()
+        {
+            // Do nothing - test progress marker
+        }
+        
+        [SetUp]
+        public void BeforeTest()
+        {
+            s_rowCount = 10;
+            _recordBatch = new RecordBatch.Builder()
+                .Append("Col_Int32", false, col => col.Int32(array => array.AppendRange(Enumerable.Range(1, s_rowCount))))
+                .Build();
+            _chunk = new ArrowResultChunk(_recordBatch);
+        }
+        
+        [Test]
+        public void TestExtractCell()
```

Comment:
can we name the test providing more information what we are really testing? also in other test methods